src/Futhark/CodeGen/ImpGen/Multicore.hs

```diff
@@ -40,19 +40,20 @@ gccAtomics = flip lookup cpu
         (Or Int64, Imp.AtomicOr Int64)
       ]
 
--- | Compile the program.
+-- | Compile the program. The boolean indicates whether dynamic scheduling may be used.
```

Comment:
Make a type for this.  `data MCOpts = NoDynamicScheduling | YesDynamicScheduling` (or whatever).  [Booleans suck.](http://sboosali.github.io/mirror/boolean-blindness.html)